src/Symfony/Component/Validator/Tests/Password/PasswordStrengthEstimatorTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Symfony\Component\Validator\Tests\Password;
+
+use PHPUnit\Framework\TestCase;
+use Stringable;
+use Symfony\Component\Validator\Constraints\PasswordStrength;
+use Symfony\Component\Validator\Password\PasswordStrengthEstimator;
+use Symfony\Component\Validator\Tests\Constraints\Fixtures\StringableValue;
+
+class PasswordStrengthEstimatorTest extends TestCase
+{
+    /** @dataProvider getPasswords */
+    public function testEstimateStrength(string|Stringable $password, int $expectedStrength): void
```

Comment:
```suggestion
    public function testEstimateStrength(string|Stringable $password, int $expectedStrength)
```

Reply:
@OskarStark I'm used to type all the functions even the ones with no return (void) => what is the benefit of not typing it?

Reply:
@yalit While I personally agree with you, the change suggested by @OskarStark reflects the code style on this repository.

Reply:
ok. noted. I updated it